logic/broadcaster.js

```diff
@@ -103,27 +103,13 @@ class Broadcaster {
 	 */
 	getPeers(params, cb) {
 		params.limit = params.limit || this.config.peerLimit;
-		params.broadhash = params.broadhash || null;
-		params.normalized = false;
-
-		const originalLimit = params.limit;
-		modules.peers.list(params, (err, peers) => {
-			if (err) {
-				return setImmediate(cb, err);
-			}
-
-			if (originalLimit === constants.maxPeers) {
-				library.logger.info(
-					[
-						'Broadhash consensus now',
-						modules.peers.getLastConsensus(),
-						'%',
-					].join(' ')
-				);
-			}
-
-			return setImmediate(cb, null, peers);
-		});
+		const peers = library.logic.peers.listRandomConnected(params);
+		library.logger.info(
```

Comment:
Please check if that log will not be spammy after removing condition.

Reply:
I checked on localhost, it seems OK, it's only once per broadcast (once for all peers), also it's info log level.